extensions/2.0/Vendor/PTC_onshape_metadata/schema/gltf.PTC_onshape_metadata.schema.json

```diff
@@ -5,6 +5,16 @@
     "description": "glTF extension for specifying IDs and visibility.",
     "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
     "properties": {
+        "documentId" : {
+            "type" : "string",
+            "minLength": 1,
```

Comment:
The minimum documentId/elementId size Onshape uses is 24, the specification currently has it set to 1. Is this by design?

Reply:
Nope I just wasn't aware of the minimum, pushing a fix 🙂